### PR TITLE
 Yurtctl revert adds the function of deleting yurt app manager.

### LINKED ISSUE #392: Yurtctl revert adds the function of deleting yurt app manager.
**What would you like to be added**:
 Yurtctl revert adds the function of deleting yurt app manager.

**Why is this needed**:
Perfect function.

**others**
/kind feature
development function.

### PR DESCRIPTION
**Pull request by yanyhui**
/kind feature

#### What this PR does / why we need it:
dds the function of deleting yurt app manager.
#### Which issue(s) this PR fixes:

Fixes #392

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/yurtctl/util/kubernetes/util.go [hunk 1]
```diff
@@ -359,6 +358,52 @@ func CreateCRDFromYaml(clientset *kubernetes.Clientset, yurtAppManagerClient dyn
 	return nil
 }
 
+func DeleteCRD(clientset *kubernetes.Clientset, yurtAppManagerClientSet dynamic.Interface, res string, name string) error {
+	var err error
+	decoder := yamlutil.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(constants.YurtAppManagerNodePool)), 10000)
```
Comment (Peeknut) [inline pkg/yurtctl/util/kubernetes/util.go:363]: Why use `constants.YurtAppManagerNodePool`? As this func `DeleteCRD` is also called when delete 
 `UnitedDeploymentcrd` in func `removeYurtAppManager`, the parameter also can be `constants.YurtAppManagerUnitedDeployment`.

**Comment by rambohe-ch** [comment]
@yanyhui Thank you for submit the pull request.
please fix the golangci-lint error.

File: pkg/yurtctl/cmd/revert/revert.go [hunk 2]
```diff
@@ -320,6 +335,116 @@ func removeYurtTunnelServer(client *kubernetes.Clientset) error {
 	return nil
 }
 
+func removeYurtAppManager(client *kubernetes.Clientset, yurtAppManagerClientSet dynamic.Interface) error {
+	// 1. remove the Deployment
+	if err := client.AppsV1().
+		Deployments(constants.YurttunnelNamespace).
+		Delete(context.Background(), constants.YurtAppManagerComponentName,
+			metav1.DeleteOptions{}); err != nil && !apierrors.IsNotFound(err) {
+		return fmt.Errorf("fail to delete the daemonset/%s: %s",
```
Comment (rambohe-ch) [inline pkg/yurtctl/cmd/revert/revert.go:344]: daemonset --> deployment

**Comment by rambohe-ch** [comment]
@yanyhui please fix the conflicted files.

**Comment by yanyhui** [comment]
to late

**Pull request closed without merging**